testsuite/MDAnalysisTests/analysis/test_nuclinfo.py

```diff
@@ -35,134 +34,162 @@
 from MDAnalysisTests.datafiles import NUCL
 
 
-class TestNuclinfo(TestCase):
-    def setUp(self):
-        self.u = mda.Universe(NUCL)
+@pytest.fixture(scope='module')
+def u():
+    return mda.Universe(NUCL)
 
-    def tearDown(self):
-        del self.u
 
-    def test_wc_pair_1(self):
-        val = nuclinfo.wc_pair(self.u, 1, 2, seg1='RNAA', seg2='RNAA')
+def test_wc_pair_1(u):
+    val = nuclinfo.wc_pair(u, 1, 2, seg1='RNAA', seg2='RNAA')
+    assert_almost_equal(val, 4.449, decimal=3)
 
-        assert_almost_equal(val, 4.449, decimal=3)
 
-    def test_wc_pair_2(self):
-        val = nuclinfo.wc_pair(self.u, 22, 23, seg1='RNAA', seg2='RNAA')
-        assert_almost_equal(val, 4.601, decimal=3)
+def test_wc_pair_2(u):
+    val = nuclinfo.wc_pair(u, 22, 23, seg1='RNAA', seg2='RNAA')
+    assert_almost_equal(val, 4.601, decimal=3)
 
-    def test_minor_pair_1(self):
-        val = nuclinfo.minor_pair(self.u, 3, 17, seg1='RNAA', seg2='RNAA')
-        assert_almost_equal(val, 16.649, decimal=3)
 
-    def test_minor_pair_2(self):
-        val = nuclinfo.minor_pair(self.u, 20, 5, seg1='RNAA', seg2='RNAA')
-        assert_almost_equal(val, 4.356, decimal=3)
+def test_minor_pair_1(u):
```

Comment:
`test_minor_pair_1` and `test_minor_pair_2` can be grouped in a parametrized `test_minor_pair`.